code/modules/food_and_drinks/recipes/drinks_recipes.dm

```diff
@@ -458,7 +458,6 @@
 
 /datum/chemical_reaction/drink/eggnog
 	results = list(/datum/reagent/consumable/ethanol/eggnog = 15)
-	required_reagents = list(/datum/reagent/consumable/ethanol/rum = 5, /datum/reagent/consumable/cream = 5, /datum/reagent/consumable/eggyolk = 5)
```

Comment:
did you intend to replace this line with something?

Reply:
uhhh pretend you didn't see that